DICOM/src/Microsoft.Health.Anonymizer.Common/Models/DateTimeGlobalSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Anonymizer.Common
```

Comment:
`.Common.Models` would be more consistent with others.

Reply:
👍  Good catch! @BoyaWu10